publications-2019.bib

```diff
@@ -13,4 +13,20 @@ @Article{Basak2019a
   url       = {https://www.researchgate.net/profile/Anup_Basak/publication/326847352_Finite_element_procedure_and_simulations_for_a_multiphase_phase_field_approach_to_martensitic_phase_transformations_at_large_strains_and_with_interfacial_stresses/links/5baae99da6fdccd3cb733c04/Finite-element-procedure-and-simulations-for-a-multiphase-phase-field-approach-to-martensitic-phase-transformations-at-large-strains-and-with-interfacial-stresses.pdf},
 }
 
+@article{ZHANG2019634,
+  title = "A high-order and interface-preserving discontinuous Galerkin method for level-set reinitialization",
+  journal = "Journal of Computational Physics",
+  volume = "378",
+  pages = "634 - 664",
+  year = "2019",
+  issn = "0021-9991",
+  doi = "10.1016/j.jcp.2018.11.029",
+  url = "http://www.sciencedirect.com/science/article/pii/S0021999118307733",
+  author = "Jiaqi Zhang and Pengtao Yue",
+  keywords = "Hamilton–Jacobi equation, Numerical flux, Second-derivative limiter, Weighted local projection, Moving contact line",
+  abstract = "A high-order numerical method for interface-preserving level-set reinitialization is presented in this paper. In the interface cells, the gradient of the level-set function is determined by a weighted local projection scheme and the missing additive constant is determined such that the position of the zero level set is preserved. In the non-interface cells, we compute the gradient of the level-set function by solving a Hamilton–Jacobi equation as a conservation law system using the discontinuous Galerkin method, following the work by Hu and Shu [SIAM J. Sci. Comput. 21 (1999) 660–690]; the missing constant is then recovered by the continuity of the level-set function while taking into account the characteristics. To handle highly distorted initial conditions, we develop a hybrid numerical flux that combines the Lax–Friedrichs flux and the penalty flux. Our method is stable for non-trivial test cases and handles singularities away from the interface very well. When derivative singularities are present on the interface, a second-derivative limiter is designed to suppress the oscillations. At least (N+1)th order accuracy in the interface cells and Nth order in the whole domain are observed for smooth solutions when Nth degree polynomials are used. Two dimensional test cases are presented to demonstrate superior properties such as accuracy, long-term stability, interface-preserving capability, and easy treatment of contact lines. We also show some preliminary results on the pinch-off process of a pendant drop, where topological changes of the fluid interface are involved. Our method is readily extendable to three dimensions and adaptive meshes."
```

Comment:
One of these hyphens is probably a dash. For simplicity, I suggest just removing the abstract entirely.